### ISSUE TITLE
Party member was counted twice somehow > Fiery Gryphon dealt double damage to all

### ISSUE THREAD

**Issue by lapidaisy** [HabitRPG/habitica#3041]
Our party is currently on our third round of the Fiery Gryphon quest. A member of our party (todefenestrate, below) was having difficulty getting her iPhone app to sync with the server to record her dailies. Then somehow on cron, the damage from those (not actually) missed dailies was not only inflicted on all of us not just once, but twice, resulting in in three deaths and the loss of a special item we don't know how to manually restore.

The messages on our party page:

> todefenestrate attacks Fiery Gryphon for 0.0 damage, Fiery Gryphon attacks party for 27.8 damage. -15 hours ago
> todefenestrate attacks Fiery Gryphon for 0.8 damage, Fiery Gryphon attacks party for 22.3 damage. -17 hours ago

TLDR: damage from party member's missed dailies during a quest hit us twice and we want our items back.

**Comment by deilann** [comment]
The special items should now be recoverable, just like other items, according to @lefnire closing that ticket.

Let's follow this here: https://github.com/HabitRPG/habitrpg/issues/2805

You can also restore anything else that was lost via Settings > Site > Fix Character Values. Give yourself any gold needed to repurchase items.
